app/components/paperShow/refCitedTab/refCitedTab.scss

```diff
@@ -13,7 +13,7 @@
     background-color: white;
     border-bottom: 1px solid $gray8;
     margin: 0;
-    z-index: 101;
+    z-index: 99;
```

Comment:
is it okay with paper show's elements? (before / after scroll)

Reply:
Yes, I checked it and it worked without problems.
refCitedTab stick -> ok
ref / cited paper layout -> ok
pdf viewer -> ok